tests/Integration/Adapter/Presenter/Product/ProductLazyArrayTest.php

```diff
@@ -91,6 +91,7 @@ class ProductLazyArrayTest extends TestCase
         'new' => 0,
         'pack' => 0,
         'out_of_stock' => OutOfStockType::OUT_OF_STOCK_DEFAULT,
+        'customizable' => 0,
```

Comment:
question: does this property have an impact on your test or did you add it for consistency with a real case scenario?

Reply:
@matthieu-rolland it was needed for a method which checks for enabling add to cart button